auto-value-parcel/src/main/java/com/ryanharter/auto/value/parcel/Parcelables.java

```diff
@@ -252,6 +265,25 @@ else if (type.equals(SIZEF))
     return block.build();
   }
 
+  public static CodeBlock writeValueWithTypeAdapter(FieldSpec adapter, AutoValueParcelExtension.Property property, ParameterSpec out) {
+    CodeBlock.Builder block = CodeBlock.builder();
+
+    if (property.nullable()) {
+      block.beginControlFlow("if ($N() == null)", property.methodName);
+      block.addStatement("$N.writeInt(1)", out);
```

Comment:
Convention when representing booleans as ints say `1 == true`, so this seems backwards to me.  If the object exists, I would expect a 1, and a 0 otherwise.